my-app/src/components/specialisation/specialisation.css

```diff
@@ -0,0 +1,63 @@
+.search-container{
```

Comment:
![image](https://github.com/Arjun-Girish/FYP-InteractiveCourseMapper/assets/87962390/0ffd2a8e-b9c4-4106-aabc-71936533bb21)
This should stretch across the entire page

Reply:
maybe keep it like this to match the styling from the previous page and the page after